### PR TITLE
Bug 1917585: Clean up olm i18n and catch some missed translations.

### PR DESCRIPTION
**Pull request by TheRealJon**
Remove unused i18n namespaces and add some missed translations in the OperatorHub subscribe page.

### CODE REVIEW

**Comment by spadgett** [comment]
```
ERROR in ./packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx
Module not found: Error: Can't resolve 'packages/console-shared/src' in '/go/src/github.com/openshift/console/frontend/packages/operator-lifecycle-manager/src/components/operator-hub'
```

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx [hunk 1]
```diff
@@ -589,12 +609,14 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
               <div className="form-group">
                 <fieldset>
                   <Popover
-                    headerContent={<div>Update Channel</div>}
-                    bodyContent={<div>The channel to track and receive the updates from.</div>}
+                    headerContent={<div>{t('olm~Update Channel')}</div>}
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx:612]:
```suggestion
                    headerContent={<div>{t('olm~Update channel')}</div>}
```

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx [hunk 2]
```diff
@@ -589,12 +609,14 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
...
+                    bodyContent={
+                      <div>{t('olm~The channel to track and receive the updates from.')}</div>
+                    }
                   >
                     <h5 className="co-required co-form-heading__popover">
                       <Button variant="plain" className="co-form-heading__popover-button">
-                        Update Channel
+                        {t('olm~Update Channel')}
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx:619]:
```suggestion
                        {t('olm~Update channel')}
```

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx [hunk 3]
```diff
@@ -610,7 +632,7 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
               </div>
               <div className="form-group">
                 <fieldset>
-                  <h5 className="co-required">Installation Mode</h5>
+                  <h5 className="co-required">{t('olm~Installation Mode')}</h5>
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx:635]:
```suggestion
                  <h5 className="co-required">{t('olm~Installation mode')}</h5>
```

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx [hunk 4]
```diff
@@ -660,22 +682,24 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
               <div className="form-group">
                 <fieldset>
                   <Popover
-                    headerContent={<div>Approval Strategy</div>}
+                    headerContent={<div>{t('olm~Approval Strategy')}</div>}
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx:685]:
```suggestion
                    headerContent={<div>{t('olm~Approval strategy')}</div>}
```

File: frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx [hunk 5]
```diff
@@ -660,22 +682,24 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
...
-                      <div>The strategy to determine either manual or automatic updates.</div>
+                      <div>
+                        {t('olm~The strategy to determine either manual or automatic updates.')}
+                      </div>
                     }
                   >
                     <h5 className="co-required co-form-heading__popover">
                       <Button variant="plain" className="co-form-heading__popover-button">
-                        Approval Strategy
+                        {t('olm~Approval Strategy')}
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx:694]:
```suggestion
                        {t('olm~Approval strategy')}
```

**Review by spadgett — COMMENTED** [review]
Thanks, @TheRealJon. We should either open a Bugzilla to track this change or associate it with a story.

**Review by spadgett — APPROVED** [review]
/lgtm
/hold

**Comment by spadgett** [comment]
Hey @TheRealJon, CONSOLE-2371 is already marked as done... I'd probably open a BZ if we missed some strings.

**Comment by spadgett** [comment]
/lgtm cancel
Looks like you need to run `yarn i18n`

**Comment by TheRealJon** [comment]
@spadgett 

> /lgtm cancel
> Looks like you need to run `yarn i18n`

@spadgett Oops, yep forgot that. Fixed now.

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
@TheRealJon Needs rebase :/

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**